packages/http-server-csharp/src/lib/utils.ts

```diff
@@ -755,3 +801,630 @@ export function getCSharpStatusCode(entry: HttpStatusCodesEntry): string | undef
       return undefined;
   }
 }
+
+export function isEmptyResponseModel(program: Program, model: Type): boolean {
+  if (model.kind !== "Model") return false;
+  return model.properties.size === 1 && isStatusCode(program, [...model.properties.values()][0]);
```

Comment:
Bit of an edge case but an empty model (no properties) should also return `true` here right? That's valid since `@typespec/http` will choose a default statusCode if one doesn't exist on the return type.

e.g. `op foo(): {}`